systems/primitives/encoder.h

```diff
@@ -0,0 +1,86 @@
+#pragma once
+
+#include <memory>
+
+#include <drake/common/eigen_types.h>
+#include <drake/multibody/plant/multibody_plant.h>
+
+#include "systems/framework/output_vector.h"
+
+#include "drake/common/drake_copyable.h"
+#include "drake/systems/framework/leaf_system.h"
+
+namespace dairlib {
+namespace systems {
+
+#define DRIVE_FILTER_NB 9
+#define JOINT_FILTER_NB 4
+#define JOINT_FILTER_NA 3
+
+static int drive_filter_b[DRIVE_FILTER_NB] = {2727, 534, -2658, -795, 72,
```

Comment:
We should not be defining Cassie-specific values in a general purpose system like this